tensor_genn/algorithms/relu_ann.py

```diff
@@ -87,29 +96,30 @@ def convert(self, tf_model):
 
         return self.g_model
 
-    def evaluate(self, X, y=None, single_example_time=350.):
+    def evaluate(self, X, y=None):
```

Comment:
Why is ``X`` upper-case and ``y`` lower case?

Reply:
That's the general convention that I've noticed for data (X) and labels (y). Should I change them to the same case?

Reply:
I think this convention with capital X an small y is quite commonly used in ML. I would recommend to keep it.

Reply:
I did not know that! In that case, keep it